receiver/datadogreceiver/receiver.go

```diff
@@ -8,6 +8,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/datadogreceiver/internal/translator"
```

Comment:
the linter would fail on this

Reply:
can you tell us why?

Reply:
We split the imports into three sections: stdlib, third party dependencies, and local packages. I believe the CI failure should point to the command to run to fix it. I think we have a make target to fix it as well (make fmt, IIRC). Can you check if this is part of the contributing guidelines?

Reply:
Ah yes I think ci lints this indeed

I dont see this being part of the contributing guidelines:
- https://github.com/open-telemetry/opentelemetry-collector/blob/main/CONTRIBUTING.md
- https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md